lib/blinkist/config/adapters/aws_ssm_adapter.rb

```diff
@@ -0,0 +1,56 @@
+require_relative "adapter"
+require "aws-sdk-ssm"
+
+module Blinkist
+  class Config
+    class AwsSsmAdapter < Adapter
```

Comment:
What's your opinion on the following:

```ruby
class AwsSsmAdapter < Adapter
  prepend ValueCaching
end

module ValueCaching
  def initialize(*args)
    super(*args)
    @__cache = {}
  end

  def get(key, default=nil, scope: nil)
    @__cache[key, scope] ||= super
  end

  protected
  
  def prefetch(key, value, scope: nil)
    @__cache[key, scope] = value
  end
end
```

This could be used also in soon to be deprecated Diplomat adapter.

BTW. There is no spec which says "if you query the the same thing twice, we will fetch only once in background"